cli/src/action/circuit/mod.rs

```diff
@@ -166,7 +166,12 @@ impl Action for CircuitProposeAction {
                 .map(ToOwned::to_owned)
                 .or_else(|| std::env::var(SPLINTER_REST_API_URL_ENV).ok())
                 .unwrap_or_else(|| DEFAULT_SPLINTER_REST_API_URL.to_string());
-            let key = args.value_of("key").unwrap_or("./splinter.priv");
+
+            let key = args
+                .value_of("key")
+                .map(String::from)
+                .unwrap_or_else(whoami::username);
```

Comment:
We should move the use of whoami::username to the cylinder function for loading keys (see other comment about loading key via cylinder function).

Reply:
Fixed